ur_robot_driver/resources/ros_control.urscript

```diff
@@ -1,20 +1,20 @@
 {{BEGIN_REPLACE}}
 
-steptime = get_steptime()
+global steptime = get_steptime()
 
 textmsg("steptime=", steptime)
-MULT_jointstate = {{JOINT_STATE_REPLACE}}
+global MULT_jointstate = {{JOINT_STATE_REPLACE}}
```

Comment:
I think, that one can skip the `global` keyword.

Reply:
You are right, i removed the statement